packages/core/src/languages/solidity/predeploys.ts

```diff
@@ -334,7 +336,10 @@ export const initializeSphinx = async (
     SphinxRegistryABI,
     owner
   )
-  const sphinxManagerV1Address = getSphinxManagerV1Address(Number(chainId))
+  const sphinxManagerV1Address = getSphinxManagerImplAddress(
```

Comment:
nit: I think we can remove V1

Reply:
Looks like there's a few remaining instances of "V1" in the TypeScript/Solidity logic